### PR TITLE
Aavechan/stablecoin IR update

### PR DESCRIPTION
**Pull request by Rozengarden**

### CODE REVIEW

**Comment by marczeller** [comment]
with updated polygon V2, both TL & Chaos labs proposal are in sync.

File: diffs/AaveV2Polygon_StablecoinIRUpdates_20240424_before_AaveV2Polygon_StablecoinIRUpdates_20240424_after.md
Comment (brotherlymite) [inline diffs/AaveV2Polygon_StablecoinIRUpdates_20240424_before_AaveV2Polygon_StablecoinIRUpdates_20240424_after.md:1]: I would remove this file as it is not being generated by any payloads currently

File: src/20240424_Multi_StablecoinIRUpdates/StablecoinIRUpdates.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+- [Snapshot](Direct-to-AIP)
```
Comment (brotherlymite) [inline src/20240424_Multi_StablecoinIRUpdates/StablecoinIRUpdates.md:92]:
```suggestion
- Snapshot: Direct-to-AIP
```

File: diffs/AaveV2Polygon_StablecoinIRUpdates_20240424_before_AaveV2Polygon_StablecoinIRUpdates_20240424_after.md
Comment (Rozengarden) [reply diffs/AaveV2Polygon_StablecoinIRUpdates_20240424_before_AaveV2Polygon_StablecoinIRUpdates_20240424_after.md:1]: done

**Pull request merged**